### ISSUE TITLE
Request for Nand Version Cubian

### ISSUE THREAD

**Issue by nocolour** [cubieplayer/Cubian#394]
Since the nand install from SD have problem. Why not create nand version cubian for us!

Thanks!

**Comment by Daniel-Medeiros** [comment]
Yes please that would be great for my cubietruck we promise we will help to seed thank you!

**Comment by amadeous** [comment]
I had a lot of trouble installing Cubian on nand with my CubieTruck..
It would be just amazing to just flash it to the nand with LiveSuit (for example) !
